tests/queries/0_stateless/02561_temporary_table_sessions.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+SESSION_ID_A="$RANDOM$RANDOM$RANDOM"
+SESSION_ID_B="$RANDOM$RANDOM$RANDOM"
+
+# Create temporary table and insert in SESSION_ID_A
+${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}&session_id=${SESSION_ID_A}" -d 'CREATE TEMPORARY TABLE table_merge_tree_02561 (id UInt64, info String) ENGINE = MergeTree ORDER BY id'
+${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}&session_id=${SESSION_ID_A}" -d "INSERT INTO table_merge_tree_02561 VALUES (1, 'a'), (2, 'b'), (3, 'c')"
+
+# Select from SESSION_ID_B
+${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}&session_id=${SESSION_ID_B}" -d "SELECT * FROM table_merge_tree_02561" | tr -d '\n' | grep -F 'UNKNOWN_TABLE' > /dev/null && echo "OK"
+
+# Create temporary table, insert and select in SESSION_ID_B
+${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}&session_id=${SESSION_ID_B}" -d 'CREATE TEMPORARY TABLE table_merge_tree_02561 (id UInt64, info String) ENGINE = MergeTree ORDER BY id'
+${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}&session_id=${SESSION_ID_B}" -d "INSERT INTO table_merge_tree_02561 VALUES (1, 'd'), (2, 'e'), (3, 'f')"
+${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}&session_id=${SESSION_ID_B}" -d "SELECT * FROM table_merge_tree_02561"
```

Comment:
Maybe check that the SELECT returned the expected values (d, e, f)? (same in l. 24)

Reply:
Robert, please explain. On l. 21:  curl sends data to server and then it displays results of SELECT in std out and which is compared with 02561_temporary_table_sessions.reference. It's quite elegant/compact way. This way we check what SELECT returns  (d, e, f).

Reply:
Sure, I missed that. No need for further checks, thanks.

Reply:
OK